rejected/codecs.py

```
@@ -0,0 +1,304 @@
+            and content_type == AVRO_DATUM_MIME_TYPE
+            and message_type
+        ):
+            schema = await self._avro_schema(message_type)
+            return fastavro.schemaless_reader(io.BytesIO(body), schema)
+
+        if content_type == 'application/json':
+            return _load_json(body)
+        if umsgpack and content_type == 'application/msgpack':
+            return _load_msgpack(body)
+        if content_type in PICKLE_MIME_TYPES:
+            return pickle.loads(body)
```

Ruff (0.15.7)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)